src/commcare_cloud/ansible/roles/elasticsearch/templates/systemd/elasticsearch.service.j2

```diff
@@ -7,8 +7,10 @@ After=network-online.target
 [Service]
 Type=simple
 Environment=ES_HOME={{ elasticsearch_home }}
-Environment=CONF_DIR={{ elasticsearch_conf_dir}}
 Environment=ES_PATH_CONF={{ elasticsearch_conf_dir}}
+{% if elasticsearch_version <= '5.6.16'%}
+Environment=CONF_DIR={{ elasticsearch_conf_dir}}
+{% endif %}
```

Comment:
Just trying to follow along. Why don't we need to define `CONF_DIR` for ES 6?

Reply:
Its not longer supported `ES_PATH_CONF` is all we need. 
https://www.elastic.co/guide/en/elasticsearch/reference/6.8/breaking-changes-6.0.html#_literal_conf_dir_literal_is_no_longer_supported